Admin/BaseFieldDescription.php

```diff
@@ -422,10 +422,14 @@ public function getMappingType()
      * @param string $property
      *
      * @return string
+     *
+     * @deprecated Deprecated since version 3.x. Use \Doctrine\Common\Inflector::camelize() instead.
      */
     public static function camelize($property)
     {
-        return Container::camelize($property);
+        @trigger_error('camelize() is deprecated since version 3.x and will be removed in 4.0. Use \Doctrine\Common\Inflector::camelize() instead.', E_USER_DEPRECATED);
```

Comment:
Done, also fixed the target from `\Doctrine\Common\Inflector` to `\Doctrine\Common\Inflector\Inflector`